content/doc/developer/security/resources/_auth_flow_schema.svg

```diff
@@ -0,0 +1,2394 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I add the source of the SVG as it could ease the future maintenance in case we change the flow. This file is not supposed to be served to the visitor.